packages/mint-components/src/components/sqm-big-stat/useBigStat.tsx

```diff
@@ -978,6 +1006,10 @@ export const queries: {
     label: "User Stat",
     query: userStatsQuery,
   },
+  payoutBalance: {
+    label: "Payout Balance",
```

Comment:
These labels probably need to be translateable?

Reply:
these are actually translatable via slots